projects/table-builder/src/lib/enums/filterTypes.ts

```diff
@@ -22,6 +22,7 @@ export enum FilterType {
     StringIn = 'Is In',
     Or = 'Or',
     And = 'And',
+    In = 'In',
```

Comment:
There already is an `'In'` . Likm mir in kup that there cannot be duplicate strings because they are used for something.

Reply:
For example 
```
export const StringFilterMap:FilterToFiltersMap = {
    [FilterType.StringEquals]: [FilterType.StringEquals],
    [FilterType.StringContains]: [FilterType.StringContains],
    [FilterType.StringDoesNotContain]: [FilterType.StringDoesNotContain],
    [FilterType.StringStartWith]: [FilterType.StringStartWith],
    [FilterType.StringEndsWith]: [FilterType.StringEndsWith],
    [FilterType.IsNull]: [FilterType.IsNull],
    [FilterType.StringIn]: [FilterType.StringIn],
};```

is using that string value